packages/template-block/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+module.exports = {
+    extends: ['@frontify/eslint-config-react'],
```

Comment:
```suggestion
    extends: ['@frontify/eslint-config-react', 'plugin:jsx-a11y/recommended'],
```

We just added it for all blocks, would be nice to have it here also

Reply:
Thanks for noticing! Added 👌